CMakeLists.txt

```diff
@@ -36,10 +36,14 @@ if(LSQPACK_XXH)
     target_sources(ls-qpack PRIVATE deps/xxhash/xxhash.c)
     set(LSQPACK_DEPENDS "")
 else()
-	find_package(PkgConfig REQUIRED)
-	pkg_check_modules(XXH REQUIRED IMPORTED_TARGET libxxhash)
-	target_link_libraries(ls-qpack PUBLIC PkgConfig::XXH)
-	set(LSQPACK_DEPENDS "libxxhash")
+    find_package(xxHash CONFIG)
+    if(NOT xxHash_FOUND)
+        find_package(PkgConfig REQUIRED)
+        pkg_check_modules(LS_QPACK_XXH REQUIRED IMPORTED_TARGET libxxhash GLOBAL)
```

Comment:
I added the `LS_QPACK_` prefix here because of the side effects of `GLOBAL` when ls-qpack is used as a subproject.